src/generated/client/apis/TimebankApi.ts

```diff
@@ -28,8 +28,8 @@ import {
 
 export interface TimebankControllerGetEntriesRequest {
     personId: string;
-    before?: Date;
-    after?: Date;
+    before?: string;
+    after?: string;
```

Comment:
Why string instead of date?

Reply:
should be
`
type: string
format: date
`
in the swagger, but there are some limitation of the nest.js as it can only define type but not format.
previously it was like this but there is no such date type in swagger so I modified the backend to use type string and do validation there.
`
type: date
`